infrastructure/pom.xml

```diff
@@ -231,6 +231,12 @@
             <artifactId>caffeine</artifactId>
             <version>2.8.5</version>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-inline</artifactId>
+            <version>3.6.0</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
The parent pom has the dependency. You can just bump up the version here: https://github.com/CorfuDB/CorfuDB/blob/master/pom.xml#L134

Reply:
We have discussed this offline and came to an agreement.